wandb/sdk/launch/launch_add.py

```diff
@@ -28,11 +29,56 @@ def launch_add(
     version: Optional[str] = None,
     docker_image: Optional[str] = None,
     params: Optional[Dict[str, Any]] = None,
-) -> "public.QueuedJob":
+    resource_args: Optional[Dict[str, Any]] = None,
+    cuda: Optional[bool] = None,
+) -> "public.QueuedRun":
+    """Enqueue a W&B launch experiment.
+
+    Arguments:
+    uri: URI of experiment to run. A wandb run uri or a Git repository URI.
+    job: string reference to a wandb.Job eg: wandb/test/my-job:latest
```

Comment:
is this supposed to be 1 or the other? can that be clarified?

Reply:
Yep, added to the main description above. Also passing in more than one raises an error indicating this issue